classes/Context.php

```diff
@@ -347,6 +348,12 @@ public function getTranslator()
             return $this->translator;
         }
 
+        $kernel = SymfonyContainer::getInstance();
+        if (!is_null($kernel)) {
+            $this->translator = $kernel->getContainer()->get('translator');
```

Comment:
You need to change `$kernel` into `$container` and then replace `$kernel->getContainer()` by `$container`